app/assets/stylesheets/frontend/helpers/_worldwide-organisation-header.scss

```diff
@@ -21,15 +21,18 @@
     margin-top: $gutter-one-third;
 
     dl {
-      @extend %govuk-body-s;
-      margin: 0;
+      @include govuk-text-colour;
+      @include govuk-font($size: 16);
+      @include govuk-responsive-margin(4, "bottom");
```

Comment:
I'm not sure but I think this may be over-riding the margin set to 0 below. 

We've gained extra whitespace at the top of this page:
https://www.integration.publishing.service.gov.uk/world/organisations/british-consulate-izmir
It's caused by extra padding at the bottom of the dl originally set to 0 on a selector of #whitehall-wrapper .worldwide-organisation-header .metadata dl

Reply:
Fixed this by [removing the page header padding](https://github.com/alphagov/whitehall/pull/6240/commits/5286f78753eea388d0a0554e33748d7d0b9dba84). I prefer <dl> elements to have a consistent margin bottom and keep the spacing in only one direction as much as possible.